packages/itwinui-react/src/styles.js/classes.mjs

```diff
@@ -2,6 +2,22 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import styles from './styles.module.css';
 
-export default styles;
+// Side-effect import to ensure CSS file is generated even though the classes are not used.
+import './styles.module.css';
+
+// This proxy converts `iui-` to `_iui3-` in class names with very little code.
+// This is more efficient than exporting the entire mapping of original-to-transformed classes.
+export default new Proxy(
+  {},
+  {
+    get(_, prop) {
+      if (typeof prop === 'string' && prop.startsWith('iui-')) {
+        return prop.replace('iui-', '_iui3-');
+      }
+    },
+    has(_, prop) {
+      return typeof prop === 'string' && prop.startsWith('iui-');
```

Comment:
Will `iui-foobar` also return `true` since it startsWith `iui-`?

Reply:
yes

Reply:
Is this alright? If so, why?

Reply:
it's fine. the `iui-` prefix belongs to us, we will never call it with unexpected strings

the only way this gets used is through the `polymorphic` utility (which is used in every component). we transform anything passed into `className`; this could include `iui-foorbar` passed by user, but they should not be doing that.

the only way to "fix" this would be to revert to the approach before this PR, which negates any bundle size benefits.